### PR TITLE
WIP: Index concepts + Kubernetes architecture 

### PR DESCRIPTION
**Pull request by perriea**
Translate:
* [x] Index concepts
* [ ] Kubernetes architecture pages
   * [x] Cloud Controller
   * [ ] Master-Node Communication
   * [ ] Node

### CODE REVIEW

**Comment by yastij** [comment]
@perriea - can you sign the CLA ?

**Comment by perriea** [comment]
> @perriea - can you sign the CLA ?

Done a minute ago :)

**Comment by yastij** [comment]
/hold 

As this is a WIP

File: content/fr/docs/concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Vue d'ensemble
+
+Pour utiliser Kubernetes, vous utilisez *les objets de l'API Kubernetes* pour décrire l'état souhaité *de votre cluster*: quelles applications ou autres charges de travail que vous souhaitez exécuter, quelles images de conteneur elles utilisent, le nombre de réplicas, les ressources réseau et disque souhaitées mettre à disposition, et plus encore. Vous définissez l'état souhaité en créant des objets à l'aide de l'API Kubernetes, généralement via l'interface de ligne de commande, `kubectl`. Vous pouvez également utiliser l'API Kubernetes directement pour interagir avec le cluster et définir ou modifier l'état souhaité.
+
+Une fois que vous avez défini l'état souhaité, le *Kubernetes Control Plane* permet de faire en sorte que l'état actuel du cluster corresponde à l'état souhaité. Pour ce faire, Kubernetes effectue automatiquement diverses tâches, telles que le démarrage ou le redémarrage de conteneurs, la mise à l'échelle du nombre de réplicas d'une application donnée, etc. Le plan de contrôle Kubernetes comprend un ensemble de processus en cours d'exécution sur votre cluster:
```
[the remark this answers is not in the document]
Comment (perriea) [reply content/fr/docs/concepts/_index.md]: I agree with that, in the other documents I'm going to push tonight, I kept the original names :)

File: content/fr/docs/concepts/_index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: Concepts
+main_menu: true
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+La section Concepts vous aide à mieux comprendre les composants du système Kubernetes et ses abstractions qu'il utilise pour représenter votre cluster. Elle vous aide également à mieux comprendre le fonctionnement de Kubernetes.
```
[the remark this answers is not in the document]
Comment (perriea) [reply content/fr/docs/concepts/_index.md]: "Kubernetes" is really used too much in en doc ...
That's why I changed the sentence a little.

[the remark this answers is not in the document]
Comment (yastij) [reply content/fr/docs/concepts/_index.md]: I agree we should make it smooth enough. Referring to Kubernetes too much without using «il » pronoun is a bit odd

[the remark this answers is not in the document]
Comment (perriea) [reply content/fr/docs/concepts/_index.md]: understood

File: content/fr/docs/concepts/_index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Vue d'ensemble
+
+Pour utiliser Kubernetes, vous utilisez *les objets de l'API Kubernetes* pour décrire l'état souhaité *de votre cluster*: quelles applications ou autres charges de travail que vous souhaitez exécuter, quelles images de conteneur elles utilisent, le nombre de réplicas, les ressources réseau et disque souhaitées mettre à disposition, et plus encore. Vous définissez l'état souhaité en créant des objets à l'aide de l'API Kubernetes, généralement via l'interface de ligne de commande, `kubectl`. Vous pouvez également utiliser l'API Kubernetes directement pour interagir avec le cluster et définir ou modifier l'état souhaité.
```
Comment (lledru) [inline content/fr/docs/concepts/_index.md]: add "," after *de votre cluster*
switch "charges de travail" with: "workloads (charge de travail)" as we discuss yesterday
...et disques **que vous souhaitez** mettre a disposition...

File: content/fr/docs/concepts/_index.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+* [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+* [Deployment](/docs/concepts/workloads/controllers/deployment/)
+* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/)
+* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+* [Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+
+## Plan de contrôle de Kubernetes
+
+Les différentes parties du plan de contrôle Kubernetes, telles que les processus Kubernetes Master et kubelet, déterminent la manière dont Kubernetes communique avec votre cluster. Le plan de contrôle conserve un enregistrement de tous les objets Kubernetes du système et exécute des boucles de contrôle continues pour gérer l'état de ces objets. À tout moment, les boucles de contrôle du plan de contrôle répondent aux modifications du cluster et permettent de faire en sorte que l'état actuel de tous les objets du système corresponde à l'état souhaité que vous avez poussé.
```
Comment (lledru) [inline content/fr/docs/concepts/_index.md]: suggest: "Control Plane"

File: content/fr/docs/concepts/_index.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Par exemple, lorsque vous utilisez l'API Kubernetes pour créer un objet Deployment, vous fournissez un nouvel état souhaité au système. Le plan de contrôle Kubernetes enregistre la création de cet objet et exécute vos instructions en lançant les applications requises tout en les planifiant sur les nœuds de cluster de manière à ce que l'état actuel du cluster corresponde à l'état souhaité.
+
+### Master Kubernetes
+
+
+Le master Kubernetes est responsable du maintien de l'état souhaité pour votre cluster. Lorsque vous interagissez avec Kubernetes, par exemple en utilisant l'interface de ligne de commande `kubectl`, vous communiquez avec le master Kubernetes de votre cluster.
+
+> Le "master" fait référence à un ensemble de processus gérant l’état du cluster. En règle générale, tous les processus sont exécutés sur un seul nœud du cluster. Ce nœud est également appelé master. Celui-ci peut également être répliqué pour la disponibilité et la redondance.
```
Comment (lledru) [inline content/fr/docs/concepts/_index.md]: redondance or résilience?

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
+---
+title: Concepts sous-jacents à Cloud Controller Manager
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Le concept de cloud controller manager (CCM, à ne pas confondre avec le binaire) a été créé à l'origine pour permettre au code de providers cloud et au noyau Kubernetes d'évoluer indépendamment les uns des autres. Le cloud controller manager fonctionne aux côtés d'autres composants principaux, tels que le Kubernetes controller manager, l'API server et le scheduler. Il peut également être démarré en tant qu’addon Kubernetes, auquel cas il s’applique au-dessus de Kubernetes.
```
Comment (lledru) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: suggest: "il s'applique" -> "tourne"

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+
+La conception du cloud controller manager repose sur un mécanisme de plugin qui permet aux nouveaux providers cloud de s'intégrer facilement à Kubernetes à l'aide de plugins. Des mesures sont en place pour l'intégration de nouveaux providers cloud sur Kubernetes et pour la migration de l'ancien modèle vers le nouveau modèle CCM.
```
Comment (lledru) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: suggest: fournisseurs de clouds

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+4. Si un node ne répond plus, vérifiez que l'instance n'a pas ete suprimée sur votre cloud provider.
+Si le node a été supprimé du nuage, supprimez l'objet Kubernetes Node.
+
+#### Route controller
+
+Le Route controller est responsable de la configuration appropriée des routes réseau dans le cloud afin que les conteneurs situés sur différentes node du cluster Kubernetes puissent communiquer entre eux. Le Route controller ne s'applique qu'aux clusters Google Compute Engine.
+
+#### Service Controller
+
+Le Service controller est chargé d'écouter les événements de création, de mise à jour et de suppression de service. Sur la base de l'état actuel des services dans Kubernetes, il configure les équilibreurs de charge cloud (tels qu'un ELB ou Google LB) pour refléter l'état des services dans Kubernetes. De plus, cela garantit que les services de base des services pour les équilibreurs de charge dans le cloud sont à jour.
```
Comment (lledru) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: suggest: load balancer

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+
+Le PersistentVolumeLabels controller a été créé spécifiquement pour le CCM; c'est-à-dire qu'il n'existait pas avant la création du CCM. Cela a été fait pour déplacer la logique de labelisation des PV dans le serveur API Kubernetes (il s'agissait d'un admission controller) vers le CCM. Il ne fonctionne pas sur le KCM.
+
+### 2. Kubelet
+
+Le Node controller contient les fonctionnalités de Kubelet dépendantes du cloud. Avant l’introduction du CCM, la sous-unité était responsable de l’initialisation de nodes avec des paramètres spécifiques au cloud, tels que les adresses IP, les étiquettes de région/zone et les informations de type d’instance. L’introduction du CCM a déplacé cette opération d’initialisation de Kubelet au CCM.
+
+Dans ce nouveau modèle, Kubelet initialise un node sans paramètres spécifiques au cloud. Cependant, il ajoute un tag taint au node nouvellement créé, ce qui le rend non modifiable jusqu'à ce que le CCM initialise le node avec les informations spécifiques au cloud. Il supprime ensuite ce tag taint.
```
Comment (lledru) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: suggest tag -> label, or :"cette teinte"

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+
+Le concept de cloud controller manager (CCM, à ne pas confondre avec le binaire) a été créé à l'origine pour permettre au code de providers cloud et au noyau Kubernetes d'évoluer indépendamment les uns des autres. Le cloud controller manager fonctionne aux côtés d'autres composants principaux, tels que le Kubernetes controller manager, l'API server et le scheduler. Il peut également être démarré en tant qu’addon Kubernetes, auquel cas il s’applique au-dessus de Kubernetes.
+
+La conception du cloud controller manager repose sur un mécanisme de plugin qui permet aux nouveaux providers cloud de s'intégrer facilement à Kubernetes à l'aide de plugins. Des mesures sont en place pour l'intégration de nouveaux providers cloud sur Kubernetes et pour la migration de l'ancien modèle vers le nouveau modèle CCM.
+
+Ce document traite les concepts du cloud controller manager et donne des détails sur ses fonctions associées.
+
+Voici l'architecture d'un cluster Kubernetes sans le cloud controller manager:
+
+![Pre CCM Kube Arch](/images/docs/pre-ccm-arch.png)
```
Comment (sftim) [inline content/fr/docs/concepts/architecture/cloud-controller.md]:
```suggestion
 ![Architecture sans CCM](/images/docs/pre-ccm-arch.png)
```

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+
+Dans le diagramme précédent, Kubernetes et le provider cloud sont intégrés via plusieurs composants différents:
+
+* Kubelet
+* controller manager Kubernetes
+* server API Kubernetes 
+
+Le CCM consolide toute la logique dépendante du cloud des trois composants précédents pour créer un point d’intégration unique avec le cloud. La nouvelle architecture avec le CCM se présente comme il suit:
+
+![CCM Kube Arch](/images/docs/post-ccm-arch.png)
```
Comment (sftim) [inline content/fr/docs/concepts/architecture/cloud-controller.md]:
```suggestion
![Architecture avec CCM](/images/docs/post-ccm-arch.png)
```

File: content/fr/docs/concepts/_index.md (hunk 5 above)
Comment (perriea) [reply content/fr/docs/concepts/_index.md]: In en doc : "The master can also be replicated for availability and redundancy."
Redundancy = redondance.

File: content/fr/docs/concepts/architecture/cloud-controller.md (hunk 7 above)
Comment (perriea) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: Do you say that in your work ? I have always use: "cloud", "cloud provider" or "provider cloud"

Comment (perriea) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: Do you say that in your work ? I have always use: "cloud", "cloud provider" or "provider cloud"

File: content/fr/docs/concepts/architecture/cloud-controller.md (hunk 9 above)
Comment (perriea) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: same as "fournisseurs de clouds"

Comment (perriea) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: label yes

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+
+## Composants du CCM
+
+Le CCM rompt avec certaines fonctionnalités du gestionnaire de contrôleurs Kubernetes (KCM) et les exécute en tant que processus séparé. Plus précisément, il sépare les contrôleurs du KCM qui dépendent du cloud. Celui-ci suis les boucles du contrôleur dépendantes du cloud :
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: processus séparés (au pluriel)

Comment (feloy) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: Celui-ci suit

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+ * Route controller
+ * Service controller
+
+Dans la version 1.9, le CCM exécute les contrôleurs suivants de la liste précédente:
+
+* Node controller
+* Route controller
+* Service controller
+
+En outre, il exécute un autre contrôleur appelé PersistentVolumeLabels controller. Celui-ci est responsable de la définition des labels par zone et région sur volumes persistant créé dans les cloud ​​GCP et AWS.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: volumes persistants créés

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+Le Volume controller a été arbitrairement choisi pour ne pas faire partie de CCM. En raison de la complexité du processus et des efforts déployés pour supprimer la logique de volume spécifique au provider, il a été décidé que le Volume controller ne serait pas déplacé vers le CCM.
+{{< /note >}}
+
+Le plan initial de prise en charge des volumes à l'aide de CCM consistait à utiliser des volumes Flex pour prendre en charge des volumes pouvant être pluggable. Toutefois, un projet concurrent appelé CSI est en cours de planification pour remplacer Flex.
+
+En tenant compte de cette dynamique, nous avons décidé de mettre en place une mesure de seuil intermédiaire jusqu’à ce que le CSI soit prêt.
+
+## Fonctions du CCM
+
+Le CCM hérite des fonctionnalitées des composants de Kubernetes qui dépendent eux même d'un provider cloud. Cette section est structurée en fonction de ces composants.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: fonctionnalités

Comment (feloy) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: eux-mêmes

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 15]
```diff
@@ -0,0 +1,264 @@
...
+* PersistentVolumeLabels controller
+
+#### Node controller
+
+Le Node controller est responsable de l'initialisation d'un node en obtenant des informations sur les nodes s'exécutant dans le cluster dans le cadre d'un provider cloud. Le node controller exécute les fonctions suivantes:
+
+1. Initialisez un node avec des labels de zone/région spécifiques au cloud.
+2. Initialisez un node avec des paramètres d'instance spécifiques au cloud, par exemple, le type et la taille.
+3. Obtenez les adresses réseau et le nom d'hôte du node.
+4. Si un node ne répond plus, vérifiez que l'instance n'a pas ete suprimée sur votre cloud provider.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: supprimée

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 16]
```diff
@@ -0,0 +1,264 @@
...
+Si le node a été supprimé du nuage, supprimez l'objet Kubernetes Node.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: nuage ? c'est le seul endroit où ce terme remplace cloud

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 17]
```diff
@@ -0,0 +1,264 @@
...
+
+#### Route controller
+
+Le Route controller est responsable de la configuration appropriée des routes réseau dans le cloud afin que les conteneurs situés sur différentes node du cluster Kubernetes puissent communiquer entre eux. Le Route controller ne s'applique qu'aux clusters Google Compute Engine.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: sur différents nodes

File: content/fr/docs/concepts/architecture/cloud-controller.md (hunk 8 above)
Comment (feloy) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: suppression de services.

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 18]
```diff
@@ -0,0 +1,264 @@
...
+
+Le route controller écoute la création d'objets Node et configure les routes de manière appropriée. Cela nécessite un accès aux objets Node.
+
+v1/Node:
+
+- Get
+
+### Service controller
+
+Le service controller écoute les creations, les mises à jour, et les suppressions d'objets de service, puis configure les noeuds ainsi que les services de manière appropriée.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: créations

File: content/fr/docs/concepts/architecture/cloud-controller.md [hunk 19]
```diff
@@ -0,0 +1,264 @@
...
+
+Pour accéder aux services, il faut un droit list et watch. Pour mettre à jour les services, un droit d'accès patch et update est necessaire.
```
Comment (feloy) [inline content/fr/docs/concepts/architecture/cloud-controller.md]: nécessaire

File: content/fr/docs/concepts/architecture/cloud-controller.md (hunk 7 above)
Comment (lledru) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: at work i use: "CSP" :), but in a french translated website, "fournisseur de cloud" looks good no? what do you think folks?

File: content/fr/docs/concepts/architecture/cloud-controller.md (hunk 9 above)
Comment (lledru) [reply content/fr/docs/concepts/architecture/cloud-controller.md]: label is slightly different no? it's more accurate to use "teinte" there i believe... but i could be wrong

**Comment by remyleone** [comment]
@perriea est ce que tu peux résoudre les conflits et rebase ?

**Comment by perriea** [comment]
@remyleone le contenu est quasi couvert par d'autres PR. Je reouvrirai une nouvelle pour proposer d'autres traductions.

**Pull request closed without merging**